src/components/core/GeoChart.tsx

```diff
@@ -24,6 +25,7 @@ export function renderLegend(props: IChartLegendProps): React.ReactElement {
 }
 
 export interface ICoreGeoChartProps extends ICommonChartProps, IDataSourceProviderInjectedProps {
+    legendPostion: string;
```

Comment:
`legendPostion` should be optional, because we already have default value for it